### DISCUSSION TITLE
Gather similar false detections

### DISCUSSION THREAD

**Discussion by jchretien** [blakeblackshear/frigate#12683] [Ideas]
It would be great if Frigate+ could re-evaluate existing detections whenever we submit new images. Using the updated model, it should review previous detections to see if any might no longer be valid. It could then group these detections together and ask for my confirmation on whether they should still be considered false positives.

**Comment by blakeblackshear** [comment]
What are you trying to accomplish?

**Reply by jchretien** [reply]
It would be nice if the events who becomes false positive would be deleted automatically after we taught model.

**Reply by blakeblackshear** [reply]
Deleted in Frigate+? If you delete those images in Frigate+, then the next model wouldn't be trained on those false positives and it would "forget".

Once you have submitted an image to Frigate+, there is no dependency on your local Frigate instance. You can delete it from your machine right away.

**Reply by jchretien** [reply]
No not from Frigate+.

But I do understand what I want is probably not that simple because there's 2 steps to submit to Frigate+. First we Send to Frigate+. Secondly we need to visit Frigate+ webpage, "Add (w)" and "Verify & Save".

What I hope was possible is within Frigate, in a single step, Send to Frigate+ the verified image.

Then the model loaded in my Frigate PC would be trained right away with the new image. The UI would then lists all the past detections that would no longer create a detection with the new model. And ask me to verify those to make sure they actually correspond to the false detection I just submitted. After I approved I would have the option to delete all those events from Frigate at once.

That process would help the user to have a instant feedback of the training, speedup the Events backlog cleanup.

I understand maybe it would be too much resource intensive to re-check several past detections, but I hope you get my "user" point of view on this.

That process could be done in background and user would have a notification with past false detections he can approve whenever he wants.
